Packs/Jira/ReleaseNotes/1_2_11.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- Reverted the change that was made in version *1.2.9* since it was not Jira Server compatible.  
+- Added the *assignee_id* argument to the ***jira-create-issue***, and the ***jira-edit-issue*** commands, which enables user assignment to an issue using the user's Account ID in order to support the changes in the Jira Cloud API.
+- Added the ***jira-get-id-by-attribute*** command, which searches and retrieves the *Account ID* for a given user's attribute.
```

Comment:
```suggestion
- Reverted a change that was made in version *1.2.9* since it was not compatible with Jira Server.  
- Added the *assignee_id* argument to the ***jira-create-issue***, and the ***jira-edit-issue*** commands, which enables user assignment to an issue using the user's Account ID in order to support changes in the Jira Cloud API.
- Added the ***jira-get-id-by-attribute*** command, which searches and retrieves the *Account ID* for a given user's attribute.
```